storage/database/interface.go

```diff
@@ -29,16 +31,22 @@ const (
 	LevelDB   DBType = "LevelDB"
 	BadgerDB         = "BadgerDB"
 	MemoryDB         = "MemoryDB"
-	DynamoDB         = "DynamoDB"
+	DynamoDB         = "DynamoDBS3"
 	ShardedDB        = "ShardedDB"
 )
 
-func (db DBType) IsValid() bool {
-	switch db {
-	case LevelDB, BadgerDB, MemoryDB, DynamoDB:
-		return true
+// ToValid converts DBType to a valid one.
+// If it is unable to convert, "" is returned.
+func (db DBType) ToValid() DBType {
```

Comment:
How about changing this to `IsValid` function which returns boolean value?

Reply:
This is to receive `dbtype` values regardless of uppercases or lowercases. ("leveldb", "levelDB", "LevelDB" and so on is all allowed.)
Could you suggest a better way of doing this? :)

Reply:
I understood. Thanks for your explanation!